Automattic/wp-calypso
# Theme Tiers: Enable in all Calypso envs

Related to:
- https://github.com/Automattic/dotcom-forge/issues/5387

## Proposed Changes

* Enabled theme tiers in all Calypso envs

## Testing Instructions
* Run `https://github.com/Automattic/dotcom-forge/issues/5387


* Checkout this branch locally.
* Run this command: `yarn feature-search themes/tiers`
* You should see that the feature is only enabled in development.

![Screenshot 2024-01-30 at 15 52 24](https://github.com/Automattic/wp-calypso/assets/1989914/5551248e-b81f-4adc-850b-e9ad31b31e58)


## Pre-merge Checklist
